src/main/fc/fc_rc.c

```diff
@@ -259,9 +265,14 @@ FAST_CODE uint8_t processRcInterpolation(void)
 }
 
 #ifdef USE_RC_SMOOTHING_FILTER
-uint8_t calcRcSmoothingCutoff(float avgRxFrameRate, float filterBaseline)
+uint16_t calcRcSmoothingCutoff(float avgRxFrameRate, float filterBaseline, bool pt1)
 {
-    return lrintf(filterBaseline / (avgRxFrameRate / RC_SMOOTHING_FILTER_AUTO_MS));
+    float cutoff = filterBaseline / (avgRxFrameRate / RC_SMOOTHING_FILTER_AUTO_MS);
+    if (pt1) { // Convert to a PT1 cutoff (2pi * cutoff)^2 / 500 / 2pi;
+        const float twoPi = 2.0f * M_PIf;
+        cutoff = (sq(cutoff * twoPi) / 500) / twoPi;
```

Comment:
Actually get rid of angular frequencies by  factoring 2pi out. That way 500 will become 80

Reply:
I can simplify the formula but I did this intentionally to show that we were using `ω_critical = 500`.  I thought in this case it made more sense from a self-documentation perspective considering that this function is only called twice and during initialization so performance isn't really a concern.

Reply:
I don't think what's the intention, what's the superiority of `500 rad/s` vs `80 Hz` when both are take arbitrarily.